src/main/resources/META-INF/resources/primefaces/treetable/treetable.css

```diff
@@ -1,4 +1,4 @@
-.ui-treetable table {
+fi.ui-treetable table {
```

Comment:
why there is `fi` ? :thinking:

Reply:
Thjat is really strange.  That is not in my local file and I can't see it in my commits.

Reply:
Even weird I see it in the commit but it is not in my local file.